scripts/testbucket-project-store.py

```
@@ -0,0 +1,131 @@
+        # from. A script that wrote `split` fields itself would be asserting
+        # the answer it was supposed to be evidence for.
+        with open(args.out_events, "w") as fh:
+            for pkg in sorted(units):
+                row = units[pkg]
+                for name in sorted(row.get("tests", {})):
+                    fh.write(json.dumps({"Action": "pass", "Package": pkg,
+                                         "Test": name, "Elapsed": row["tests"][name]}) + "\n")
+                fh.write(json.dumps({"Action": "pass", "Package": pkg,
+                                     "Elapsed": row["seconds"]}) + "\n")
+    total = sum(u["seconds"] for u in units.values())
+    print("projected %d packages to %s: total %.1fs (%.1f min) -> %s"
```

Ruff (0.16.2)

[warning] Use format specifiers instead of percent format

(UP031)